pkg/controllers/nodeclaim/disruption/drift_test.go

```diff
@@ -124,7 +124,12 @@ var _ = Describe("Drift", func() {
 	It("should detect static drift before cloud provider drift", func() {
 		cp.Drifted = "drifted"
 		nodePool.Annotations = lo.Assign(nodePool.Annotations, map[string]string{
-			v1beta1.NodePoolHashAnnotationKey: "123456789",
+			v1beta1.NodePoolHashAnnotationKey:        "test-123456789",
+			v1beta1.NodePoolHashVersionAnnotationKey: "v1",
```

Comment:
```suggestion
			v1beta1.NodePoolHashVersionAnnotationKey: v1beta1.NodePoolHashVersion
```

We should match this to whatever the constant is currently set to; otherwise, we are going to have to keep updating this value whenever there is a change in this version